### PR TITLE
check system connections before machine init

### PR DESCRIPTION
**Pull request by baude**
we should exit early if a system connection name exists with the name of the proposed podman machine (i.e. podman-machine-default).

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM

File: cmd/podman/machine/init.go [hunk 1]
```diff
@@ -138,6 +138,17 @@ func initMachine(cmd *cobra.Command, args []string) error {
 	if _, err := provider.LoadVMByName(initOpts.Name); err == nil {
 		return fmt.Errorf("%s: %w", initOpts.Name, machine.ErrVMAlreadyExists)
 	}
+
+	cfg, err := config.ReadCustomConfig()
+	if err != nil {
+		return err
+	}
+
+	//	check if a system connection already exists
```
Comment (Luap99) [inline cmd/podman/machine/init.go:147]: shouldn't you check for both connection names, i.e. also the -root one?

**Review by Luap99 — APPROVED** [review]
LGTM

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Pull request merged**